src/util.ts

```diff
@@ -156,11 +156,12 @@ export function parseFilePrelude(text: string): Prelude {
 		}
 		if (line.match(/^(\s)*import(\s)+\(/)) {
 			ret.imports.push({ kind: 'multi', start: i, end: -1, pkgs: [] });
-		}
-		if (line.match(/^(\s)*import(\s)+[^\(]/)) {
+		} else if (line.match(/^\s*import\s+"C"/)) {
```

Comment:
Is C the only import that falls under this pseudo category?

Reply:
Yes. "C" is the only one in this category (psuedo import whose placement in the import affects the correctness/functionality)